src/System.Runtime.Extensions/tests/CustomAttributesTestApp/CustomAttributesTestApp.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why is the PR adding .csproj files?

Reply:
My environment was dirty, that file belongs to another branch. Just went over the diff, that was the only mistake. Thanks for pointing out.